facebookincubator/facebook-for-woocommerce
# Add the connection screen

# Summary

Adds the dedicated Connection screen.

### Story: [CH 55207](https://app.clubhouse.io/skyverge/story/55207)
### Release: #1355 

## UI Changes

- Unconnected state: https://cloud.skyver.ge/04uPXXnx
- Connected state: https://cloud.skyver.ge/YEu199w9

## QA

Note - a few followup stories not part of this PR:
- Hook up the [disconnect URL](https://app.clubhouse.io/skyverge/story/54055)
- Add the final [Facebook image assets](https://app.clubhouse.io/skyverge/story/55216)
- Revamp settings [acceptance tests](https://app.clubhouse.io/skyverge/story/55220)

1. If already connected, simulate disconnect by temporarily clearing your access token
1. Go to WooCommerce -> Facebook
    - [x] The settings page is displayed as in the UI image above
    - [x] The "Get Started button links to the FBE OAuth flow
1. Restore your access token or connect again via the flow
1. Reload the settings page
    - [ ] The settings page is displayed as in the UI image above
    - [x] The "Create Ad" button links to Facebook in a new window
    - [x] The "Manage" button links to Facebook in a new window (will error out)
1. Toggle debug logging (the only real setting to test)
1. Save changes
    - [x] Your choice was persisted

- [x] `codecept run integration APITest` passes
- [x] `codecept run integration API` passes

## Before merge
